bridge/slack/handlers.go

```diff
@@ -176,7 +181,7 @@ func (b *Bslack) handleMessageEvent(ev *slack.MessageEvent) (*config.Message, er
 	return rmsg, nil
 }
 
-func (b *Bslack) handleStatusEvent(ev *slack.MessageEvent, rmsg *config.Message) bool {
+func (b *Bslack) handleStatusEvent(ev *slack.MessageEvent, rmsg *config.Message) (doneProcessing bool) {
```

Comment:
Why are we introducing a named-return? It does not appear to be set anywhere.

Reply:
Sorry, I thought this was how golang documents returns for skimming functions. If this is never done, happy to revert

Reply:
Please remove the named return.